src/render/painter.js

```diff
@@ -400,24 +454,30 @@ class Painter {
         }
 
         gl.linkProgram(program);
-        assert(gl.getProgramParameter(program, gl.LINK_STATUS), gl.getProgramInfoLog(program));
+        assert(gl.getProgramParameter(program, gl.LINK_STATUS), (gl.getProgramInfoLog(program): any));
 
         const numAttributes = gl.getProgramParameter(program, gl.ACTIVE_ATTRIBUTES);
         const result = {program, numAttributes};
 
         for (let i = 0; i < numAttributes; i++) {
             const attribute = gl.getActiveAttrib(program, i);
-            result[attribute.name] = gl.getAttribLocation(program, attribute.name);
+            if (attribute) {
+                result[attribute.name] = gl.getAttribLocation(program, attribute.name);
+            }
```

Comment:
Alternative:

```js
if (!attribute) assert(false);
result[attribute.name] = gl.getAttribLocation(program, attribute.name);
```

It's a little weird, but (a) closer to what we really want to say (which is presumably just `assert(attribute)`) and easier to migrate if/when flow supports more flexible guards, (b) unassertify will make this into `if(!attribute) {}` which minification would remove, and (c) two LOC instead of 3 😏